### PR TITLE
Counter

### PR DESCRIPTION
**Pull request by estelle**
Added counter() function support BCD
Has been supported in all browsers, starting with IE8. Know this from using this feature forever.
Ticket #1188
page: https://developer.mozilla.org/en-US/docs/counter

### CODE REVIEW

File: css/types/counter.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+  "css": {
+    "types": {
+      "attr": {
```
Comment (ddbeck) [inline css/types/counter.json]: Missed this on copy and paste?

```suggestion
      "counter": {
```

File: css/types/counter.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/counter",
+          "description": "<code>counter()</code>",
+          "support": {
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": true
```
Comment (ddbeck) [inline css/types/counter.json]: I'm going to guess that it's safe to go with the first release here

```suggestion
              "version_added": "18"
```

Comment (jpmedley) [reply css/types/counter.json]: Correct.

**Comment by vinyldarkscratch** [comment]
Hey @estelle, do you plan to follow up on this PR?

**Pull request closed without merging**